syntax = "proto3";
package example_service;
option cc_generic_services = true;

message ResultCode
{
    int32 error_code = 1;
    bytes error_msg = 2;
}

message GetfriendsListRequest
{
    uint32 user_id = 1;
}

message GetfriendsListResponse
{
    ResultCode result = 1;
    // friends array
    repeated bytes friend = 2;
}

service FriendServiceRpc
{
    rpc GetfriendsList(GetfriendsListRequest) returns(GetfriendsListResponse);
}
